### PR TITLE
Update README 

### PR DESCRIPTION
**Pull request by utkarsh006**
- solves #355 
- See [changes](https://github.com/utkarsh006/dummygram/tree/updation) here

### CODE REVIEW

**Comment by utkarsh006** [comment]
@narayan954 Kindly Review.

File: README.md [hunk 1]
```diff
@@ -1,354 +1,50 @@
-<div id="top"></div>
 <div align="center">
 
-<img  src="src/assets/app-logo.png"  alt="Dummygram"  width="250"  height="200">
```
Comment (narayan954) [inline README.md:4]: logo looked nice

File: README.md [hunk 2]
```diff
@@ -1,354 +1,50 @@
...
-      <img src="https://img.shields.io/github/issues-pr-closed/narayan954/dummygram?style=for-the-badge" />
-      <img src="https://img.shields.io/github/last-commit/narayan954/dummygram?style=for-the-badge" />
- </div>
-
----
+## DummyGram
 
+[See Demo Here](https://narayan954.github.io/dummygram/)
 </div>
+<img src="https://img.shields.io/github/repo-size/narayan954/dummygram?style=for-the-badge" />
```
Comment (narayan954) [inline README.md:7]: why did you remove rest of the badges? I think they'd still look good

File: README.md [hunk 3]
```diff
@@ -1,354 +1,50 @@
...
-   ```zsh
-   npm run cypress:open
-   ```
+||
+|---|
+|<img width="300" src="https://user-images.githubusercontent.com/79099734/189589410-ca17afb8-5855-4316-918a-054f27594809.png">|
+|<img height="300" src="https://user-images.githubusercontent.com/77617189/205454287-c5bffb68-d08f-4103-a191-02b7f54fb81f.png">|
+|<img width="300" src="https://user-images.githubusercontent.com/77617189/205456062-b28a333d-c48b-41ef-96c3-676ef2363ade.png">|
+|<img width="300" src="https://user-images.githubusercontent.com/77617189/215256338-87d294c6-c33e-4da6-8236-f8ab6fae6c4b.png">|
+|<img width="300" src="https://user-images.githubusercontent.com/77617189/219851281-59de273a-388a-4367-a7c6-39d37196ee90.png">|
```
Comment (narayan954) [inline README.md:25]: img tag needs an alt

File: README.md [hunk 4]
```diff
@@ -1,354 +1,50 @@
...
+<details><summary><h3>Open Source Programs this Repo had been a part of 😉  </h3></summary>
 
-3. Then, [test](#start-testing) your test cases:
-
-   ```zsh
-   npm run cypress:open
-   ```
+||
+|---|
+|<img width="300" src="https://user-images.githubusercontent.com/79099734/189589410-ca17afb8-5855-4316-918a-054f27594809.png">|
```
Comment (narayan954) [inline README.md:21]: this arrangement is too linear, small and not visually good

File: info.md [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+- If you want to implement a new feature, open an issue first to discuss what it'd look like .
+- If you want to contribute but are unsure to start, we have [a "good first issue" label](https://github.com/narayan954/dummygram/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) which is applied to newcomer-friendly issues. Take a look at [the full list of good first issues](https://github.com/narayan954/dummygram/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) and pick something you like!
+- Want to fix a bug or implement an agreed-upon feature? Great, jump to the [local setup instructions](#first-time-setup)!
+
+## Code of Conduct
+
+Dummygram is now part of GitHub. For code of conduct, please see [GitHub's Community Guidelines](https://help.github.com/en/github/site-policy/github-community-guidelines) and [Acceptable Use Policies](https://help.github.com/en/github/site-policy/github-acceptable-use-policies).
+
+<div  align="center"><img  height="70px"  src="https://user-images.githubusercontent.com/77617189/192942891-31b9152c-918b-4fac-af05-0ad6b1f594aa.svg"  /></div>
+
```
Comment (narayan954) [inline info.md:259]: readme-hindi is untouched, that doesn't make sense. You may remove readme-hindi

**Review by narayan954 — CHANGES_REQUESTED** [review]
You may make the main readme more appealing by letting some old parts like logo stay in place.Also I think there's some loss in information. I'll confirm it by reviewing again, but till then can you please review by yourself as well?

File: README.md (hunk 2 above)
Comment (utkarsh006) [reply README.md:7]: I think rest logos are irrelevant to show, bcz users don't have concern regarding no. of issues/PR's that are open or closed.

File: README.md (hunk 3 above)
Comment (utkarsh006) [reply README.md:25]: @narayan954 They are still being displayed from the remote source.

File: info.md (hunk 5 above)
Comment (utkarsh006) [reply info.md:259]: I followed the basic convention by renaming file names in smallcase letters.

**Comment by utkarsh006** [comment]
@narayan954 Kindly resolve conversations which are satisfying your requirements.

File: info.md (hunk 5 above)
Comment (narayan954) [reply info.md:259]: I am not talking about naming conventions... I was telling to either remove the file or edit it as well. We can't have different translations in different formats. Doesn't make sense

File: README.md (hunk 3 above)
Comment (narayan954) [reply README.md:25]: Well that doesn't excuse for an alt property

File: README.md (hunk 2 above)
Comment (narayan954) [reply README.md:7]: Still, let them stay.

File: README.md (hunk 1 above)
Comment (narayan954) [reply README.md:4]: you may put the logo back

**Comment by utkarsh006** [comment]
@narayan954 I think our visions doesn't match. I would suggest you to look at how good documentations are maintained. All the best for your future endeavors.

**Comment by narayan954** [comment]
> @narayan954 I think our visions doesn't match. I would suggest you to look at how good documentations are maintained. All the best for your future endeavors.

works!
